pkg/types/date.go

```diff
@@ -351,6 +351,18 @@ func (f gomentFormat) parseTime(s string) (TimeOfDay, error) {
 		Seconds: int64(t.Second()), Minute: int64(t.Minute()), Hour: int64(t.Hour())}, nil
 }
 
+func (f goFormat) formatTime(s string) (string, error) {
```

Comment:
func (f goFormat) formatTime(in TimeOfDay) (string,error) {
   in.ToTime().Format(string(f))
}
func (f goFormat) formatDateTime(in DateTime) (string,error) {...}
func (f goFormat) formatDate(in Date) (string,error) {...}

func (f gomentFormat) formatTime(in Time) (string,error) {
  gomentFormat(f, in. gomentTime())
}